stmhal/i2c.c

```diff
@@ -116,6 +116,16 @@ I2C_HandleTypeDef I2CHandle2 = {.Instance = NULL};
 I2C_HandleTypeDef I2CHandle3 = {.Instance = NULL};
 #endif
 
+// Default DMA parameters to pass to dma_init(); first two params are Channel
+// and Direction which get passed separately.
+// Remaining values in order are PeriphInc, MemInc, PeriphDataAlignment, MemDataAlignment,
+// Mode, Priority, FIFOMode, FIFOThreshold, MemBurst, and PeriphBurst
+STATIC const DMA_InitTypeDef i2c_dma_init = {
```

Comment:
You can use named initialisation to make it clearer: `.PeriphInc = DMA_PINC_DISABLE, .MemInc = ...`.